packages/babel-plugin-emotion/src/babel-utils.js

```diff
@@ -90,12 +90,19 @@ export function getLabel(
   filename: string
 ) {
   if (!identifierName || !autoLabel) return null
-  if (!labelFormat) return identifierName.trim()
+
+  // Valid Characters in CSS Class Names Selecter
+  // https://stackoverflow.com/questions/448981/which-characters-are-valid-in-css-class-names-selectors#449000
+  const transformedidentifierName = identifierName.replace(/[^\w-]/g, '')
```

Comment:
this should be properly camelCased, also maybe `normalizedName` (or similar) would be a better and more readable name?